src/SFML/Graphics/CMakeLists.txt

```diff
@@ -117,6 +117,12 @@ elseif(SFML_OS_IOS)
     target_link_libraries(sfml-graphics PRIVATE z bz2)
 endif()
 
+# on newer MSVC versions, inline versions of some C functions are used; for compatibility link this library
+# see https://docs.microsoft.com/en-us/cpp/porting/overview-of-potential-upgrade-issues-visual-cpp?view=msvc-160#libraries
+if(SFML_COMPILER_MSVC)
+    target_link_libraries(sfml-graphics PRIVATE legacy_stdio_definitions.lib)
+endif()
```

Comment:
Do we need some additional library check constraint, i.e. does this also compile/link with e.g. VS 2010

Reply:
Good point, according to [this page], they support legacy_stdio_definitions.lib from Visual Studio 2015 and later.

Which version is still officially supported by SFML? Also, there are a few pending PRs introducing move semantics, so they will anyway break very old versions.

Reply:
For SFML 3 it won't matter, but up until then, I'd add the following constraint as well.

`if(SFML_COMPILER_MSVC AND MSVC_VERSION GREATER_EQUAL 1900)`

Reply:
Updated.